### PR TITLE
Changed the name of Media embed button to be Media Library 

### PR DESCRIPTION
**Pull request by yasmeensalah**
please review

### CODE REVIEW

File: modules/varbase_features/varbase_media/varbase_media.install [hunk 1]
```diff
@@ -60,4 +60,9 @@ function varbase_media_update_8001() {
   $field_preview_view_mode_config_content_data = (array) Yaml::parse($field_preview_view_mode_config_content);
   $field_preview_view_mode_config->setData($field_preview_view_mode_config_content_data)->save();
 
+	// Change name of Media embed button to be “Media Library".
```
Comment (moerazem) [inline modules/varbase_features/varbase_media/varbase_media.install]: @yasmeensalah please change your editor's settings to use double spaces for tabs. No tabs allowed in Drupal coding standards :)

File: modules/varbase_features/varbase_media/varbase_media.install [hunk 2]
```diff
@@ -61,8 +61,8 @@ function varbase_media_update_8001() {
   $field_preview_view_mode_config->setData($field_preview_view_mode_config_content_data)->save();
 
   // Change name of Media embed button to be “Media Library".
-  $embed_button_config = \Drupal::service('config.factory')->getEditable('embed.button.media')
-  ->set('label','Media Library')
-  ->save();
+  $embed_button_config = \Drupal::service('config.factory')->getEditable('embed.button.media');
+  $embed_button_config->get('label')=='media' ? $embed_button_config->set('label','Media Library')
```
Comment (Natshah) [inline modules/varbase_features/varbase_media/varbase_media.install]: - Not sure why the **shorthand!** condition at this point.
- A normal **If** statement will do with the Drupal 8 standard.
- A smarter condition could include the call for the **strtolower** php function to lower the old label then compare.

**Review by Natshah — CHANGES_REQUESTED** [review]

File: modules/varbase_features/varbase_media/varbase_media.install [hunk 3]
```diff
@@ -62,7 +62,9 @@ function varbase_media_update_8001() {
 
   // Change name of Media embed button to be “Media Library".
   $embed_button_config = \Drupal::service('config.factory')->getEditable('embed.button.media');
-  $embed_button_config->get('label')=='media' ? $embed_button_config->set('label','Media Library')
-  ->save() :'';
-
+  if (strtolower($embed_button_config->get('label')) === 'media')
+  {
```
Comment (Natshah) [inline modules/varbase_features/varbase_media/varbase_media.install]: Drupal Standard coding. keep the { in the same line with the if statement

File: modules/varbase_features/varbase_media/varbase_media.install [hunk 4]
```diff
@@ -60,4 +60,11 @@ function varbase_media_update_8001() {
   $field_preview_view_mode_config_content_data = (array) Yaml::parse($field_preview_view_mode_config_content);
   $field_preview_view_mode_config->setData($field_preview_view_mode_config_content_data)->save();
 
+  // Change name of Media embed button to be “Media Library".
+  $embed_button_config = \Drupal::service('config.factory')->getEditable('embed.button.media');
+  if (strtolower($embed_button_config->get('label')) === 'media')
+  {
```
Comment (Natshah) [inline modules/varbase_features/varbase_media/varbase_media.install]: Please, keep the { in the same line with the If statement

**Comment by Natshah** [comment]
Committed to the Drupal.org first. No need to marge

**Pull request closed without merging**